Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -321,7 +321,14 @@ extension Generator {
         entity.properties = try makeInlineProperties(for: name, object: details, context: context)
             .filter { !$0.type.isVoid }
             .removingDuplicates(by: \.name) // Sometimes Swifty bool names create dups
-        entity.protocols = getProtocols(for: name, context: context)
+        
+        entity.protocols = {
+            var protocols = getProtocols(for: name, context: context)
+            let isIdentifiable = entity.properties.contains { $0.name.rawValue == "id" }
```

Comment:
Are there likely to be common cases where the `type` of the `id` property isn't `Hashable`? 

I'm wondering if it would be preferable to check if the type is either `string` or `integer`? 

Maybe its better to start with a narrower scope that we are confident of, and if there are other types, we can widen it in the future.

Reply:
Good catch, didn't think about safety. Luckily, OpenAPIKit can tell us the subset of primitive types that it will decode to, mainly based upon OpenAPI 3: 

https://github.com/mattpolzin/OpenAPIKit/blob/main/Sources/OpenAPIKit/Schema%20Object/TypesAndFormats.swift

https://github.com/mattpolzin/OpenAPIKit/blob/main/Sources/OpenAPIKit/Schema%20Conformances/SwiftPrimitiveTypes+OpenAPI.swift

This is the small list I can create with all of the hashable types like I mentioned.

Reply:
I think we can probably save ourselves some trouble by only worrying about `string`, `integer` and maybe even `number` but I doubt anybody is seriously going to use `boolean`, `array` or `object` for an `id` property and expect it to also be `Identifiable`?

WDYT?

Reply:
That seems like a safe assumption to make. We would then only have to worry about anything else if somebody requests it.

Reply:
I attempted shortly to make a general `Hashable` conformance for all 4 `TypeIdentifier` types however it's not appropriate here. We can actually safely assume that `id` is a primitive type as all of the primitive types given to us conform to `Hashable`, either in initial declaration or an extension that is also `Encodable`, which all of the objects must be since they are parsed.

I only ran into an issue when it came to determining whether a user defined type was `Hashable` as that needs to recursively do other user defined types on properties based on the context. I'm sure there are other things to think about as well.